src/layout/Grid.js

```diff
@@ -422,27 +445,35 @@
     };
 
     Grid.prototype.updateDropCells = function (dimensions, cellWidth, cellHeight) {
-        var dropCells = [];
-        if (this.designMode()) {
-            for (var rowIdx = 0; rowIdx < dimensions.height; ++rowIdx) {
-                for (var colIdx = 0; colIdx < dimensions.width; ++colIdx) {
-                    dropCells.push({ x: colIdx, y: rowIdx });
-                }
+        if(this.designMode()){
+            var c_canvas = document.createElement('canvas');
```

Comment:
No it doesn't need to... Good point, I'll modify this to only redraw the canvas when necessary.